### PR TITLE
add flag to show sbom files during a package inspect command

### LINKED ISSUE #677: Inspecting a package should let you see the SBOM information within the package
When a Zarf package is created a SBOM of the images is included within the package and is displayed when the package is deployed. Showing the SBOM information when a package is being inspected would be very useful too.

**Describe the solution you'd like**
Give users the ability to view SBOM information when they are running the `zarf package inspect` command.

### PR DESCRIPTION
**Pull request by YrrepNoj**
## Description
This PR adds an optional flag to display SBOM information when inspecting a Zarf package. This gives users the ability to see SBOM information before they run a `zarf package deploy` command.

Fixes #677

### CODE REVIEW

**Comment by jeff-mccoy** [comment]
Would you mind dropping a screenshot in the PR comments when you get a chance?

File: src/cmd/package.go [hunk 1]
```diff
@@ -129,4 +129,5 @@ func init() {
 	packageDeployCmd.Flags().StringVar(&config.DeployOptions.SGetKeyPath, "sget", "", "Path to public sget key file for remote packages signed via cosign")
 
 	packageInspectCmd.Flags().StringVar(&config.CommonOptions.TempDirectory, "tmpdir", "", "Specify the temporary directory to use for intermediate files")
+	packageInspectCmd.Flags().BoolVarP(&packager.ViewSBOM, "sbom", "s", false, "View SBOM contents while inspecting the package.")
```
Comment (Racer159) [inline src/cmd/package.go:132]: Looks like the docs were not regenerated with this flag being added

**Review by Racer159 — CHANGES_REQUESTED** [review]

File: src/cmd/package.go (hunk 1 above)
Comment (YrrepNoj) [reply src/cmd/package.go:132]: good catch, just added it now

**Review by jeff-mccoy — APPROVED** [review]
great add, txh

**Review by Racer159 — APPROVED** [review]

**Pull request merged**